App/Package/Core/Controllers/ThemeController.php

```diff
@@ -296,20 +296,21 @@ private function themeReset(): void
     #[NoReturn]
     private function adminThemeDelete(string $theme): void
     {
+        $themeName = base64_decode($theme);
         UsersController::redirectIfNotHavePermissions('core.dashboard', 'core.themes.manage');
 
-        if ($theme === ThemeManager::getInstance()->getCurrentTheme()->name() && $this->uninstallTheme($theme)) {
+        if ($themeName === ThemeManager::getInstance()->getCurrentTheme()->name() && $this->uninstallTheme($themeName)) {
```

Comment:
Pourquoi appeler 2 fois uninstallTheme ?

Fais 1 variable: $uninstallStatus = $this->uninstallTheme($themeName);

Reply:
Ah oui, je comprends que ça paraisse étrange, je m'explique 

Premier if -> Le thème est le currentTheme et il faut le désinstaller donc double check + Changement du thème par défaut par Sampler. Mais avant de mettre Sampler par défaut, on vérifie qu'on arrive bien à le désinstaller

elseif -> donc ce n'est pas le thème par défaut, mais il faut quand même le désinstaller

Reply:
Wtf c'est quoi ce bourbier

Reply:
Fixet 👨🏻‍🔧👨🏻‍🔧